CONTRIBUTING.md

```diff
@@ -27,13 +27,11 @@ into three categories:
 2. Find an issue to work on. The best way is to look for the [good first issue](https://github.com/rapidsai/cuml/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
     or [help wanted](https://github.com/rapidsai/cuml/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) labels
 3. Comment on the issue saying you are going to work on it
-4. Get familar with the developer guide relevant for you:
```

Comment:
Why remove the references to the developer guide?

Reply:
I was under the impression this developer guide was internal. Do we have a developer guide for cuML on Github?

Reply:
Yeah, Jiri Added it in https://github.com/rapidsai/cuml/blob/branch-0.7/cuML/DEVELOPER_GUIDE.md, for cython a PR from Thejaswi has one as well so we can add it when that PR is in

Reply:
I do remember that, but didn't realize it was in the cuML directory. I do agree, we should keep the references to the developer guide.